README.md

```diff
@@ -1,56 +1,23 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Greetings Microverse
 
 > Description the project.
 
 
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+-css and html
 
 
```

Comment:
## Your Branch name
![pr-title (copy)](https://user-images.githubusercontent.com/93039359/185084550-1b071e4e-1d9e-4289-8a71-51a947cc6980.png)

- [ ] Your branch name is not descriptive enough. You are using **hello-micro** which is not describing what are you doing in this branch, you can use  **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1: